core/embed/rust/src/ui/layout_caesar/component/input_methods/passphrase.rs

```diff
@@ -398,7 +398,9 @@ impl Component for PassphraseEntry {
 
         if let Some((action, long_press)) = self.choice_page.event(ctx, event) {
             match action {
-                PassphraseAction::CancelOrDelete => {
+                CancelableChoiceAction::<PassphraseAction>::Choice(
```

Comment:
```suggestion
                CancelableChoiceAction::Choice(
```
seems to work fine for me without it